lib/vtt.js

```diff
@@ -1469,4 +1478,4 @@
 
   global.WebVTT = WebVTT;
 
-}(this));
+}(this, (this.vttjs || {})));
```

Comment:
Also, if we don't mind breaking the API and want to support the browser and node, we should look into using a tool like browserify. I think it'll let us simplify some of the code related to having this work in both the browser and node.